### PR TITLE
Register for Agency

### PR DESCRIPTION
**Pull request by sondth-1037**
## Related Tickets:
- [#27902](https://edu-redmine.sun-asterisk.vn/issues/27902)

## WHAT this PR does?
- Register For Agency

## Notes Impacted Areas:
- N/A.

## Attachments:
register_agency

### CODE REVIEW

**Comment by sondth-1037** [comment]
ready

**Comment by sondth-1037** [comment]
ready

File: app/Http/Controllers/Agency/AgencyController.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        $input['role_id'] = $role->id;
+
+        $user = User::create($input);
+
+        if ($user) {
+            alert()->success(trans('contents.common.alert.title.create_account_success'), trans('contents.common.alert.message.create_account_success'));
+        } else {
+            alert()->error(trans('contents.common.alert.title.create_account_fail'), trans('contents.common.alert.message.create_account_fail'));
+        }
+        return redirect()->route('agency.getLogin');
```
Comment (huyenttm-0901) [inline app/Http/Controllers/Agency/AgencyController.php]: 1 dòng trống trc return

Comment (sondth-1037) [reply app/Http/Controllers/Agency/AgencyController.php]: em đã sửa

**Comment by sondth-1037** [comment]
ready

**Pull request merged**